source/common/upstream/BUILD

```diff
@@ -51,8 +51,14 @@ envoy_cc_library(
 
 envoy_cc_library(
     name = "cluster_manager_lib",
-    srcs = ["cluster_manager_impl.cc"],
-    hdrs = ["cluster_manager_impl.h"],
+    srcs = [
+        "cluster_manager_impl.cc",
+        "lazy_loader_impl.cc",
```

Comment:
Prefer to have `lazy_loader_impl.*` in their own `lazy_load_lib` target to keep fine grained dependencies for testing.